src/main/java/qna/domain/Answer.java

```diff
@@ -66,20 +71,31 @@ public User getWriter() {
         return writer;
     }
 
-    public boolean isDeleted() {
+    public Deleted isDeleted() {
```

Comment:
```suggestion
    public boolean isDeleted() {
```
관례적으로 is를 붙이면 반환타입은 boolean을 사용합니다.

Reply:
반영하였습니다. 😀